pipelines/config.py

```diff
@@ -95,6 +95,5 @@
     *(f for f in _os.listdir(".") if _os.path.isfile(f) and f.endswith(REFORMATTING_FILE_EXTS)),
     ".github",
     "pages",
-    "docs",
```

Comment:
Re-add this and use `--ignore-words-list="nd"` when running codespell

Reply:
Btw --ignore-words-list doesn't exist rn, that's in this pr https://github.com/codespell-project/codespell/pull/2040. You'd probably want to use --ignore-words and make a file for it rn